example/spec/parallel/lzardsBackup/LzardsBackupSpec.js

```diff
@@ -138,6 +147,99 @@ describe('The Lzards Backup Task ', () => {
     expect(backupStatus[0].statusCode).toBe(201);
```

Comment:
Add checks for `provider` and `collectionId`.

Reply:
done